optimum/onnxruntime/modeling_decoder.py

```diff
@@ -130,8 +131,11 @@ def __init__(
         self.session_outputs = {output_key.name: idx for idx, output_key in enumerate(self.session.get_outputs())}
         self.session_input_names = list(self.session_inputs.keys())
         self.session_output_names = list(self.session_outputs.keys())
-        self.key_value_input_names = [key for key in self.session_input_names if "key_values" in key]
-        self.key_value_output_names = [key for key in self.session_output_names if "key_values" in key]
+        # TODO: make this less hacky.
+        self.key_value_input_names = [key for key in self.session_input_names if (".key" in key) or (".value" in key)]
```

Comment:
did not understand it all - why is there this change?

Reply:
Because the test was not working for the past key values input / output names